src/pages/Books.js

```diff
@@ -0,0 +1,9 @@
+import classes from './Books.module.css';
+
+const Books = () => (
+  <div className={classes.books}>
+    <h1>Books Page</h1>
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] [OPTIONAL] It looks like this component is not necessary as you have the `BookList` component which renders the books